source/javascripts/circle-members.js

```diff
@@ -0,0 +1,36 @@
+var $donorJSON = 'https://membership.texastribune.org/circle-members.json';
+
+$.ajax({
+  url: $donorJSON,
+  type: 'GET',
+  dataType: 'json',
+  success: buildWall
+});
+
+
+//slugifies names for table row classes
+var slug = function(str) {
+    var $slug = '';
+    var trimmed = $.trim(str);
+    $slug = trimmed.replace(/[^a-z0-9-]/gi, '-').
+    replace(/-+/g, '-').
+    replace(/^-|-$/g, '');
+    return $slug.toLowerCase();
+};
+
+
+function buildWall(data) {
+	$.map(data, function(val, i) {
+		slugCircles = slug(i);
+		circleId = '#'+slugCircles;
+		$(circleId).append('<h3 class="circle-title">' + i + '</h3>');
+		$.map(val, function(name, index) {
+			if (index == val.length-1) {
+				$(circleId).append('<span class="circle-list">&nbsp;' + name + ' </span> ');
+			} else {
+				$(circleId).append('<span class="circle-list">&nbsp;' + name + ' <span class="yellow-star">&#9733;</span></span> ');
```

Comment:
For stars on the member levels page, the project currently uses a Font Awesome icon. It might be nice if we could stay consistent with our :star: icons throughout the project.